packages/elements/src/components/ino-chip/ino-chip.tsx

```diff
@@ -36,6 +36,11 @@ export class Chip implements ComponentInterface {
    */
   @Prop() inoIconLeading = false;
 
+  /**
+   * If enabled, appends the slotted icon to the chip label
```

Comment:
We also should mention that this flag will disable the `ino-removable` functionality

Reply:
✅